### DISCUSSION TITLE
The server is started, but I can't connect the server,The error msg is" next exited with code 0"

### DISCUSSION THREAD

**Discussion by liangxiaowucai** [reworkd/AgentGPT#1016] [Q&A]
<img width="1032" alt="image" src="https://github.com/reworkd/AgentGPT/assets/42310513/83873107-7416-48f4-857c-eba550e3918d">

**Comment by jasangill1** [comment]
Hello @liangxiaowucai Are you using docker to set up ?? Not sure what the error is would you be able to provide me with more logs?
